commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionOptionRelLocalServiceImpl.java

```diff
@@ -71,6 +71,14 @@ public CPDefinitionOptionRel addCPDefinitionOptionRel(
 			ServiceContext serviceContext)
 		throws PortalException {
 
+		CPDefinitionOptionRel cpDefinitionOptionRel =
```

Comment:
@luca-pellizzon we can't add 2 same options to a product?

Reply:
you can add more specifications but only 1 option per type